setup.py

```diff
@@ -859,7 +859,11 @@ def debug_build():
         license="HPND",
         author="Alex Clark (PIL Fork Author)",
         author_email="aclark@python-pillow.org",
-        url="http://python-pillow.org",
+        url="https://python-pillow.org/",
```

Comment:
```suggestion
        url="https://python-pillow.org",
```
Nitpick: Would you be able to leave this as it is, unless there's a reason you've changed it?